js/components/importBrowserDataPanel.js

```diff
@@ -0,0 +1,108 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
would it be possible to move this to the new directory structure?
https://github.com/brave/browser-laptop/blob/master/docs/directoryStructure.md

So for example, this would move to:
app/renderer/components

Great job on the control :smile: